src/UndoRedo.spec.ts

```diff
@@ -2,4 +2,130 @@
 declare var require: NodeRequire;
 
 import test = require('tape');
-import undoRedo from './';
+import undoRedoStack from './';
+
+interface countObj {
```

Comment:
Can we make this TypeCase or w/e the word is haha. `CountObj` and similar update on line `12`